packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -936,7 +884,9 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     return (
       this.renderTemplate('person', { person }, person.id) ||
       html`
-         <mgt-person .personDetails=${person} .fetchImage=${!this.disableImages}></mgt-person>
+         <mgt-person .personDetails=${person} .fetchImage=${!this.disableImages} .personCardInteraction=${
+        PersonCardInteraction.none
+      }></mgt-person>
```

Comment:
for readability let's split this manually to stack the attributes
```suggestion
         <mgt-person 
           .personDetails=${person} 
           .fetchImage=${!this.disableImages} 
           .personCardInteraction=${PersonCardInteraction.none}
         ></mgt-person>
```

Reply:
Fixed!